apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -63,12 +63,15 @@
 
 #ifdef CONFIG_FS_SMARTFS
 
-#if defined(CONFIG_SIDK_S5JT200_AUTOMOUNT_USERFS)
+#if defined(CONFIG_SIDK_S5JT200_AUTOMOUNT_USERFS) && !defined(CONFIG_SMARTFS_MULTI_ROOT_DIRS)
```

Comment:
1. In previous commit, you modified only artik05x,h. Not included sidk_s5jt200.
2. ```le_tc``` is not familiar to user (Samsung internally uses). Let's change it to ```testcase```.

Reply:
okay will change it and will include in sidk_s5jt200 as well.